xkill@eson57/files/xkill@eson57/xkill@eson57.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+if [ ! -e ~/.killdone ]
+then
+zenity --info --title="XKill" --text="Click on the application that you want to kill."
```

Comment:
Take a look at the other Actions that have translation support for Zenity if you would like the window to have translatable strings. Shout if you need a hand with this.

Reply:
Yes, I already thought about this, and will take a look. I just didn't think it was necessary for initial release. Still I might need help with this one to.

Reply:
This is ultimately your call. Shout if you need a hand whenever you get around to implementing it.

Reply:
I've now tested two found 'methods' and none of them worked, so I will have to do more research on this.

Reply:
I think I fixed it.

Reply:
You could simplify this for yourself by defining it once as a variable. That way, if you ever need to update it, you can just do it once.